### ISSUE TITLE
example load_from command not working

### ISSUE THREAD

**Issue by rosskarchner** [cfpb/regulations-bootstrap#6]
On my mac (not using vagrant)

```
./build_from.py ../fr-notices/annual/CFR-2012-title12-vol8-part1026.xml 12 2011-31715 15 1601
```

fails with:

```
usage: build_from.py [-h] [--generate-diffs | --no-generate-diffs]
                     [--checkpoint CHECKPOINT]
                     filename title act_title act_section
build_from.py: error: argument act_title: invalid int value: '2011-31715'
```

**Comment by rosskarchner** [comment]
(python 2.7.10, using the virtualenv built by `./regs_bootstrap.sh -b parser`)

**Comment by willbarton** [comment]
So, you don’t need to run the parser, unless you just want to (and if you do, boy do we have some work to keep you busy!). It generates the JSON that’s in regulations-stub, which is why regulations-stub exists.

That said, you need to remove the 2011-31715 altogether. Where is that line in the documentation? It needs to be changed.

—Will

**Comment by rosskarchner** [comment]
It's in the very last section of the README in this repo

**Comment by willbarton** [comment]
D'oh! Reading it as email traffic and didn't notice this was a Github notification!

So, that's fixed in #7.
